packages/select/test/suggestTests.tsx

```diff
@@ -210,6 +210,36 @@ describe("Suggest", () => {
         }
     });
 
+    describe("Uncontrolled Mode with default value", () => {
+        it("initialize the selectedItem with the defaultSelectedItem", () => {
+            const defaultSelectedItem = TOP_100_FILMS[0];
+            assert.isNotNull(defaultSelectedItem, "The selected item we test must not be null");
```

Comment:
don't bother with this assertion, it is not relevant to the actual test itself. it's always defined.